tests/trainer/test_lr_finder.py

```diff
@@ -152,6 +152,31 @@ def test_call_to_trainer_method(tmpdir):
         'Learning rate was not altered after running learning rate finder'
 
 
+def test_datamodule_parameter(tmpdir):
+    """ Test that the datamodule parameter works """
+
+    # trial datamodule
+    dm = TrialMNISTDataModule(tmpdir)
+
+    hparams = EvalModelTemplate.get_default_hparams()
+    model = EvalModelTemplate(**hparams)
+
+    before_lr = hparams.get('learning_rate')
+    # logger file to get meta
+    trainer = Trainer(
+        default_root_dir=tmpdir,
+        max_epochs=2,
+    )
+
+    lrfinder = trainer.lr_find(model, mode='linear')
```

Comment:
```suggestion
    lrfinder = trainer.tuner.lr_find(model, datamodule=dm)
```

Reply:
Oops, I remembered a short while after committing that I'd forgotten to add the ``datamodule`` parameter. That'll teach me to copy-paste code...